src/components/shared/table-wrapper/table-wrapper.module.scss

```diff
@@ -1,6 +1,9 @@
 .table-wrapper {
   width: 100%;
   overflow: auto;
+  td:nth-child(1) {
+    white-space: nowrap;
```

Comment:
@ppcano we've added this style which prevents breaks in the first column of all tables. The reason for this was to avoid the following scenario where the emoji and name of the option were on two different lines:

<img width="427" alt="Screenshot 2022-07-12 at 14 05 18" src="https://user-images.githubusercontent.com/1112428/178685530-5d2c8ff2-6922-4932-b89d-6d4f18c4309a.png">

This change will probably affect all the tables in our docs, so we're not 100% sure whether this is a good idea as it may break a table elsewhere. Do you have any other suggestions or do you think this is ok?

Reply:
It is fine for me. 

@SilencerWeb, any other proposal?